clusterctl/examples/google/addons.yaml.template

```diff
@@ -0,0 +1,205 @@
+# Note: kubectl apply the generated file after the cluster is provisioned
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: standard
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
can you add a comment of how this annotation is used

Reply:
Agreed that we should not document common annotations. Only things we add or unusual choices.